arithmetization/src/main/java/net/consensys/linea/zktracer/container/module/StatelessModule.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.container.module;
+
+import net.consensys.linea.zktracer.container.ModuleOperation;
+import net.consensys.linea.zktracer.container.stacked.StackedSet;
+import org.hyperledger.besu.evm.worldstate.WorldView;
+
+/**
+ * A {@link StatelessModule} is a {@link Module} that contains unordered {@link E} where we don't
+ * keep trace of duplicates.
+ */
+public interface StatelessModule<E extends ModuleOperation> extends Module {
+
+  StackedSet<E> operations();
```

Comment:
What is the difference between StatelessModule and StatefullModule, as I see that both have operations as a field ? I would imagine that StatelessModule doesn't have any field.

Reply:
One is using a StackedSet, the other one a StackedList

Reply:
Why do you call it StatelessModule in this case ?

Reply:
cf next comment. Renamed